internal/infrastructure/kubernetes/infra_client.go

```diff
@@ -30,7 +30,7 @@ func (cli *InfraClient) CreateOrUpdate(ctx context.Context, key client.ObjectKey
 			if kerrors.IsNotFound(err) {
 				// Create if it does not exist.
 				if err := cli.Client.Create(ctx, specific); err != nil {
-					return fmt.Errorf("for Create %v", err)
+					return errors.Wrap(err, "for Create")
```

Comment:
```suggestion
					return errors.Wrap(err, "failed to create resource")
```

Reply:
https://github.com/envoyproxy/gateway/blob/cc68246562d6b3b1010415612ccdfcf49172d400/internal/infrastructure/kubernetes/infra.go#L54-L72

the `prefix` has already existed

Reply:
```bash
2023-06-19T19:24:31.902+0800    ERROR   infrastructure  runner/runner.go:69     failed to create new infra      {"runner": "infrastructure", "error": "failed to create or update serviceaccount envoy-gateway-system/envoy-default-eg-63636f73: for Update:..",
```